book/07-git-tools/sections/credentials.asc

```diff
@@ -132,51 +132,51 @@ username=bob <3>
 There are several key features this program needs to have:
 
-. The only action we need to pay attention to is `get`; `store` and `erase` are write operations, so we'll just exit cleanly when they're received.
-. The file format of the shared-credential file is the same as that used by `git-credential-store`.
-. The location of that file is fairly standard, but we should allow the user to pass a custom path just in case.
+. Ang tanging aksyon na kailangan natin bigyang pansin ay ang `get`; `store` at `erase` ay sumusulat na mga operasyon, kaya lalabas lang tayo ng malinos kapag sila ay nakatanggap.
+. Ang file format sa ibinahagi-credential na file ay pareho na ginagamit ng `git-credential-store`.
+. Ang lokasyon sa file na iyon ay medyo karaniwan, pero dapat nating pahintulutan na ang gumagamit ay makapasa ng isang pasadyang path kung sakali.
 
-Once again, we'll write this extension in Ruby, but any language will work so long as Git can execute the finished product.
-Here's the full source code of our new credential helper:
+Muli, magsusulat tayo ng extensyon ng Ruby na ito, ngunit ang anumang wika ay gagana hangga't maipatutupad ng Git ang maaaring magsagawa ng natapos na produkto.
+Narito ang buong source code sa ating bagong credential na katulong:
 
 [source,ruby]
 --------
 include::../git-credential-read-only[]
 --------
 
-<1> Here we parse the command-line options, allowing the user to specify the input file. The default is `~/.git-credentials`.
-<2> This program only responds if the action is `get` and the backing-store file exists.
-<3> This loop reads from stdin until the first blank line is reached.
-    The inputs are stored in the `known` hash for later reference.
-<4> This loop reads the contents of the storage file, looking for matches.
-    If the protocol and host from `known` match this line, the program prints the results to stdout and exits.
+<1> Dito namin mai-parse ang command-line na mga opsyon, pumapayag sa gumagamit na magtakda ng input file. Ang default ay `~/.git-credentials`.
+<2> Ang programa ay tumutugon lamang kung ang aksyon ay `get` at ang backing-store na file ay umiiral.
+<3> Itomg loop ay nagbabasa mula sa stdin hanggang ang unang blangkong linya ay naabot.
+    Ang mga input ay na-imbak sa `known` na hash para sa reperensiya mamaya.
+<4> Ang loop ay nagbabasa ng nilalaman sa imbakan ng file, naghahanap ng mga tumutugma.
+    Kung ang protocol at host mula sa `known` na tugma sa linyang ito, ang programa ay nag-print ng mga resulta sa stdout at umiiral.
 
-We'll save our helper as `git-credential-read-only`, put it somewhere in our `PATH` and mark it executable.
-Here's what an interactive session looks like:
+I-save namin ang aming mga katulong bilang `git-credential-read-only`, ilagay ito sa isang lugar sa aming `PATH` at markahan itong executable.
+Narito ang kung ano ang interactive na sesyon na nagmumukha:
```

Comment:
change Narito ang kung ano ang interactive na sesyon na nagmumukha: to Narito kung ano ang hitsura ng isang interactive na sesyon:

Reply:
done change